### ISSUE TITLE
Using keyboard shortcut to close shortcuts listing throws error

### ISSUE THREAD

**Issue by mflorida** [swissspidy/web-stories-wp#11555]
## Bug Description

Using the <kbd>cmd</kbd><kbd>/</kbd> shortcut to _close_ the listing of keyboard shortcuts throws an error. However, after dismissing the error, using the <kbd>cmd</kbd><kbd>/</kbd> shortcut a _second_ time to close the listing works as intended.

## Expected Behaviour

The keyboard shortcuts list should close without error.

## Steps to Reproduce

Open the keyboard shortcut listing - either by clicking on the icon or using the <kbd>cmd</kbd><kbd>/</kbd> shortcut.

## Screenshots

https://user-images.githubusercontent.com/4173034/169910313-31431ba3-4c36-4562-a9f3-84438f883ee8.mov

## Additional Context

- Plugin Version: 1.21.0-alpha.0
- WordPress Version: 5.9.3
- Operating System: macOS
- Browser: Chrome 101.0.4951.64

**Comment by swissspidy** [comment]
Nice find!
